### ISSUE TITLE
Crash on bake operation in 4.1 beta and 4.2 alpha

### ISSUE THREAD

**Issue by marklio** [EatTheFuture/compify#1]
I converted a traditional motion-tracking/ foreground/background compositing scene to compify and I'm getting a blender crash when I try to "bake footage lighting". I haven't yet tried a "from scratch" scene, so I don't know whether this is a compify issue with 4.1/4.2, or some issue with the scene itself. Either way, I figured a report was useful since the scene is pretty vanilla.

I'm super unfamiliar with typical Blender crash diagnostics techniques (but super familiar with general windows crash diagnostics). What diagnostic artifacts would be useful in diagnosing? Dumps? Logs? .blend files?

**Comment by marklio** [comment]
I have validated that a "from scratch" scene doesn't crash, so this is something with my scene. I can't share it as-is, but I will try to come up with a repro .blend file.

**Comment by marklio** [comment]
I'm going to close this given the title is no longer accurate, and if someone else hits something like this, they can see that this likely isn't 4.1/4.2 causing it. I'll open another issue when I have a nice repro.

**Comment by marklio** [comment]
Just to add some notes for posterity...

I got my scene working without crashing. The things that I did that made it work:
* Removed nested collections under the Footage Geo collection - I had organized things for foreground/background that allowed me to set sub-collections as hold-out or indirect only. I collapsed these.
* I joined all my complex object hierarchies into simple meshes. I had alot of complicated objects with complex parent/child relationships with empties, etc. in my Footage Geo collection (ex. I had a bunch of tree assets that I was projecting footage onto to fake masking.)
* I removed any "shadow catcher" markings on objects (I removed this after I stopped crashing, but the compify scripts were failing with shadow catchers in the footage geo)

No idea if any of these were the first-order causes of crashing, or if there was some combo of things. Ultimately, I'd guess there's a Blender bug in there somewhere.
